src/plugins/outputPlayedSong/outputPlayedSong.cpp

```diff
@@ -15,7 +18,385 @@ constexpr GH3::QbKey IntroArtistInfoText = "intro_artist_info_text";
 static int pluginState = 0;
 
 static std::wstring songName;
+static std::wstring songNameEx = L"";
 static std::wstring songArtist;
+static std::wstring songArtistEx = L"";
+static std::wstring prefix = L"";
+static bool prefSet = false;
+
+typedef unsigned short word;
+
+static std::wstring AAnThe[] = {
+	L"A",
+	L"AN",
+	L"THE"
+};
+
+static std::wstring OtherLowercaseWords[] = {
+	L"IS",
+	L"TO",
+	L"FOR",
+	L"AND",
+	L"NOR",
+	L"BUT",
+	L"OR",
+	L"YET",
+	L"SO",
+	L"ON",
+	L"IN",
+	L"AT",
+	L"BY",
+	L"OVER",
+	L"INTO",
+	L"ONTO",
+	L"FROM",
+	L"WITH",
+	L"OF"
+};
+
+static word Lowercase[] = {
```

Comment:
Unicode transformation is overkill.  It also makes no sense to have title case transformations like the exception words above that are language-specific combined with unicode transformations